File: fatawa-server-master/TeamWork.Core/Models/Fatawa/QuestionDiscussionModel.cs
using System;

namespace TeamWork.Core.Models.Fatawa
{
    public class QuestionDiscussionModel : BaseModel
    {
        public int? SenderId { get; set; }
        public string? SenderName { get; set; }
        public string? SenderPhotoUrl { get; set; }
        public string MessageText { get; set; }
        public int QuestionId { get; set; }
        public int RepliedId { get; set; }
        public string? RepliedName { get; set; }
        public string? RepliedPhotoUrl { get; set; }
        public bool? IsRead { get; set; }
        public DateTime? DateRead { get; set; }
        public bool IsPublished { get; set; }

    }
}

File: fatawa-client-Full-update-with-new-design/src/app/infrastructure/models/project/fatawa/questionDiscussionModel.ts
export interface QuestionDiscussion {
    id: number;
    questionId: number;
    senderId: number;
    senderName?: string;
    SenderPhotoUrl?: string;
    repliedId: number;
    repliedName?: string;
    repliedPhotoUrl?: string;
    messageText: string;
    isRead?: boolean;
    createDate?: Date;
}
